joomla/joomla-cms
# [4.0] User field custom element clean up

Pull Request for Issue # .

### Summary of Changes
- Just tighten up the code a bit and cleared the console error

This might need a bit more refinement but will not happen before the new modals script

### Testing Instructions

Edit an article, try to change the author check your console and also that everything still works

### Expected result



### Actual result



### Documentation Changes Required

Nope
